sharded_queue/api.lua

```diff
@@ -319,24 +292,31 @@ function sharded_queue.statistics(tube_name)
             leader_only = true
         })
 
-    local ok, stats_collection = map('tube_statistic',
-        { tube_name = tube_name }, storages)
+    local stats_collection, err = cartridge_pool.map_call('tube_statistic',
+        {{ tube_name = tube_name }}, {uri_list=storages})
 
-    if not ok or stats_collection == nil then
+    if err then
+        error(err)
+    end
+
+    if next(stats_collection) == nil then
         log.info(stats_collection)
```

Comment:
stats_collection is empty, why we log it?

Reply:
Removed in https://github.com/tarantool/sharded-queue/pull/19/commits/936dda04875ae662190915d35a5e7c04bc4a8e34